packages/plugins/__template/package.json

```diff
@@ -0,0 +1,54 @@
+{
+  "name": "@blockly/plugin-template",
+  "version": "1.20200312.2",
+  "description": "A Blockly plugin that ...",
+  "scripts": {
+    "build": "blockly-scripts build",
+    "clean": "blockly-scripts clean",
+    "dist": "blockly-scripts build prod",
+    "lint": "blockly-scripts lint",
+    "prepublishOnly": "npm run clean && npm run prod",
+    "start": "blockly-scripts start"
+  },
+  "main": "./dist/index.js",
+  "unpkg": "./dist/index.js",
+  "author": "Blockly Team",
+  "keywords": [
+    "blockly",
+    "plugin",
+    "..."
+  ],
+  "homepage": "https://github.com/google/blockly-samples/tree/master/packages/plugins/...#readme",
+  "bugs": {
+    "url": "https://github.com/google/blockly-samples/issues"
+  },
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/google/blockly-samples.git",
+    "directory": "packages/plugins/..."
+  },
+  "license": "Apache-2.0",
+  "directories": {
+    "dist": "dist",
+    "src": "src"
+  },
+  "files": [
+    "dist",
+    "src"
+  ],
+  "devDependencies": {
+    "@blockly/dev-tools": "^0.20200313.1",
+    "@blockly/dev-scripts": "^0.20200403.0",
+    "blockly": "^3.20200123.1"
+  },
+  "peerDependencies": {
+    "blockly": ">=3.20200123.0"
```

Comment:
Might want to add a TODO or something here since people should update this depending on their package.

Reply:
pretty hard to add a TODO there and still have it be valid. This is the sort of thing that can go into a codelab / Getting started.